### ISSUE TITLE
Not updating weather/time info after connecting 

### ISSUE THREAD

**Issue by umbra2707** [GeekMagicClock/smalltv-ultra#58]
Hello! 

I got a smalltv Ultra. I got connection and location set up but it doesn't update the info on the display.

The device connects to my wifi as it should and I can reach it through a browser. I've used the code for my city an the example code but nothing happens. I even got an API Key from OpenWeather but it's the same. 

![IMG_20240510_143122508](https://github.com/GeekMagicClock/smalltv-ultra/assets/16968099/cc780713-a648-4796-8a7e-b2a1b7b9e053)

Am I doing something wrong? I'm currently using the latest version (Model: Smalltv-Ultra,Version: Ultra-V9.0.21)

**Comment by GeekMagicClock** [comment]
Yes, something is wrong with your settings, show me your settings on the web please, the city code area.

**Comment by umbra2707** [comment]
I could get it working but only on my Android phone hotspot. When I change to my WiFi router it stops working again.

**Comment by GeekMagicClock** [comment]
> I could get it working but only on my Android phone hotspot. When I change to my WiFi router it stops working again.

This should be a network issue, is there any blacklist or whitelist on your router?
